.github/workflows/ci_cd.yml

```diff
@@ -101,7 +101,19 @@ jobs:
           ANSYSLMD_LICENSE_FILE: 1055@${{ secrets.LICENSE_SERVER }}
           ANSYS_WORKBENCH_LOGGING_FILTER_LEVEL: 0
         run: |
-          xvfb-run /install/ansys_inc/v232/aisol/.workbench_lite make -C doc html
+          xvfb-run /install/ansys_inc/v232/aisol/.workbench_lite make -C doc html > doc_build_ouput.txt
```

Comment:
I think you might be only redirecting stdout here. In case of catastrophic error, your file won't capture it. Capture stderr as well